vcs/VTKPlots.py

```diff
@@ -1428,16 +1442,77 @@ def scaleLogo(self):
                 self.setLayer(self.logoRenderer, 1)
                 self.renWin.AddRenderer(self.logoRenderer)
 
-    def fitToViewport(self, Actor, vp, wc=None, geoBounds=None, geo=None, priority=None,
-                      create_renderer=False, add_actor=True):
+    def _applyTransformationToDataset(self, T, data):
+        global index
```

Comment:
We need to remove the index and debugWriteGrid - this was only for debugging purposes.